### ISSUE TITLE
Tab order is broken in the event editor

### ISSUE THREAD

**Issue by rezbyte** [tutao/tutanota#6656]
**Describe the bug**
When navigating the event editor using a keyboard, the tab gets stuck at the 'from' date field and does not move to the time selector next to it. Manually clicking past them and onto the all day checkbox reveals another problem, the tab also gets stuck the 'repeating' drop-down.

**To Reproduce**
1. Create or edit an event to cause the event editor to open
2. Try to navigate the event editor using only 'tab' and 'shift + tab'.
3. See that you get stuck at the aforementioned places.

**Expected behavior**
You should be able to tab through each interactable input starting from the cancel button and ending at the description continuously.

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser: Chromium
 - Version 122

**Comment by wrdhub** [comment]
Here is what I noticed while testing:
- tab to month controls, change month with enter. Cannot select day (tab goes to time, and arrows keys do not work)
- if there is a one day in the first/last week of the month home/end will only go to the 2nd/next to last day of the month (can test with September 2024, it has both! Dec 2024 has the 1st on its own week, and March 2025 has the 31st on its own)
